### PR TITLE
fix: #998 eslint auto fix and fix moveEndHandler will trigger that when i click the enabled element, but it is not the same with 2.4.x"

### LINKED ISSUE #998: vNext - moveEndHandler will trigger that when i click the enabled element, but it is not the same with 2.4.x
### Prerequisites

* [x] Are you running the latest version?
* [x] Are you reporting to the correct repository?
* [x] Did you perform a cursory search?

For more information, see the `CONTRIBUTING` guide.

### Description

[Description of the bug or feature]

### Steps to Reproduce
( In vNext )
1. Active LengthTool
2. Click the enabled element
3. there is a new measurement with 0 px length and i can't continue to draw the measurement

(In 2.4.x)

1. Active LengthTool
2. Click the enabled element
3. there is a new measurement
4. move on the element
5. Click the other area, then i can draw a new measurement

**Expected behavior:** [What you expected to happen]

same as 2.4.x

### PR DESCRIPTION
**Pull request by xingbofeng**
…ed element, but it is not the same with 2.4.x and eslint auto fix

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

fix the #998  and I run `npm run lint` auto fix eslint

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: src/tools/annotation/CircleRoiTool.test.js [hunk 1]
```diff
@@ -246,21 +246,44 @@ describe('CircleRoiTool.js', () => {
...
-        100, 3, 6,
-        100, 100, 100,
-        100, 4, 5,
-        100, 3, 6,
-        100, 100, 100,
-        100, 4, 5,
-        100, 3, 6
-      ];
-    };
+    external.cornerstone.getPixels = () => [
```
Comment (dannyrb) [inline src/tools/annotation/CircleRoiTool.test.js]: Can we throw an eslint/prettier ignore around this?

File: src/tools/annotation/EllipticalRoiTool.test.js [hunk 2]
```diff
@@ -16,11 +16,7 @@ jest.mock('./../../externalModules.js', () => ({
     metaData: {
       get: jest.fn(),
     },
-    getPixels: () => {
-      return [100, 100, 100,
-        100, 4, 5,
-        100, 3, 6];
-    }
+    getPixels: () => [100, 100, 100, 100, 4, 5, 100, 3, 6],
```
Comment (dannyrb) [inline src/tools/annotation/EllipticalRoiTool.test.js]: Same here? Sometimes it's easier to visualize if the shape matches how you would imagine the canvas.

File: src/tools/annotation/ProbeTool.test.js [hunk 3]
```diff
@@ -17,11 +17,10 @@ jest.mock('./../../externalModules.js', () => ({
       get: jest.fn(),
     },
     getStoredPixels: (element, x, y) => {
-      const storedPixels = [10, 20, 30,
-        40, 50, 60,
-        70, 80, 90];
+      const storedPixels = [10, 20, 30, 40, 50, 60, 70, 80, 90];
```
Comment (dannyrb) [inline src/tools/annotation/ProbeTool.test.js]: And here ^_^

File: src/tools/cursors/MouseCursor.js [hunk 4]
```diff
@@ -23,7 +23,7 @@ export default class MouseCursor {
   }
 
   /**
-   * getIconSVG - Returns an SVG of the icon only.
+   * GetIconSVG - Returns an SVG of the icon only.
```
Comment (dannyrb) [inline src/tools/cursors/MouseCursor.js]: We can probably remove the name from the beginning of the description of each of these.

1. It's already included in the JSDoc meta
2. Forcing caps on comments for the first letter technically creates a casing issue with the method's name

**Comment by dannyrb** [comment]
0. Address small comments
1. Get the deploy preview to pass
2. Make sure other tests are passing

Then I think this is good to merge.

**Comment by dannyrb** [comment]
Screenshot_20190722-064535_Chrome

It looks like some prettier errors are still breaking the build. I'll add a quick change that auto-runs prettier on committed files shortly. In the short term, you can either update these manually, or run prettier's autofix as well

**Comment by xingbofeng** [comment]
I had fixed it. @dannyrb , sorry for i forgot my pr these days......

**Pull request merged**